ONSdigital/srm-rh-ui
# 1316 Pass secret key through base config, use non default session name

# Motivation and Context

We need to configure the flask secret key for sessions to work (enabling flash messages)

# What has changed



- Pass the secret key through the base config
- Use a custom session cookie name to avoid ambiguity or clashes

# How to test?



Run the app using the base config, passing in a random secret key. Check the flash messages work when entering a UAC (invalid dummy UAC is fine).

# Links



https://trello.com/c/tlfATzmc/1316-runbook-and-configure-rh-ui-flask-session-secret-key-5
